daemon/src/lib.rs

```diff
@@ -67,6 +67,15 @@ pub const N_PAYOUTS: usize = 200;
 /// - The auto-rollover time-window (taker)
 pub const SETTLEMENT_INTERVAL: time::Duration = time::Duration::hours(24);
 
+/// Perform necessary housekeeping before actor system startup
+pub async fn perform_housekeeping(db: &SqlitePool, wallet: &Address<wallet::Actor>) -> Result<()> {
+    let mut conn = db.acquire().await?;
+
+    housekeeping::transition_non_continue_cfds_to_setup_failed(&mut conn).await?;
+    housekeeping::rebroadcast_transactions(&mut conn, wallet).await?;
+    Ok(())
+}
```

Comment:
Can we move this function to `housekeeping::new` and make the other two private?

Reply:
great idea